mmv1/templates/terraform/examples/cloudbuild_trigger_service_account.tf.erb

```diff
@@ -0,0 +1,25 @@
+resource "google_cloudbuild_trigger" "<%= ctx[:primary_resource_id] %>" {
+  trigger_template {
+    branch_name = "master"
+    repo_name   = "my-repo"
+  }
+
+  service_account = "projects/my-project/serviceAccounts/my-service-account"
```

Comment:
You'll want to reference the service account here, similar to below- the `id` field should hold the right format, I think.

Reply:
Done. Changed to `"${google_service_account.cloudbuild_service_account.id}"`.